### PR TITLE
revouninstaller: Add version 2.3.5

### PR DESCRIPTION
**Pull request by issaclin32**
supersedes #5205, #562

[RevoUninstaller](https://www.revouninstaller.com/) is a tool for uninstalling programs including a file scan for leftovers after uninstallation.

### CODE REVIEW

File: bucket/revouninstaller.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "description": "Software uninstaller with support for additional cleanup of registries/files",
+    "homepage": "https://www.revouninstaller.com/products/revo-uninstaller-free/",
+    "license": "Freeware",
+    "url": "https://www.revouninstaller.com/download-free-portable.php#/dl.zip",
+    "hash": "1637c4f962b33ffc509a367927342a7858ba8e09b8802e09fe589323e5dd0652",
+    "extract_dir": "RevoUninstaller_Portable",
+    "pre_install": [
+        "'ctrlbars.dat', 'un_report.dat', 'settings.ini', 'winapp_report.dat' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) {",
+        "        Set-Content \"$dir\\$_\" $content -Encoding Ascii",
```
Comment (rashil2000) [inline bucket/revouninstaller.json:12]: What is `$content` here?

File: bucket/revouninstaller.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "2.3.5",
+    "description": "Software uninstaller with support for additional cleanup of registries/files",
+    "homepage": "https://www.revouninstaller.com/products/revo-uninstaller-free/",
+    "license": "Freeware",
+    "url": "https://www.revouninstaller.com/download-free-portable.php#/dl.zip",
```
Comment (pratikpc) [inline bucket/revouninstaller.json:6]: 32 bit?

Comment (issaclin32) [reply bucket/revouninstaller.json:6]: It is 32-bit.

File: bucket/revouninstaller.json (hunk 1 above)
Comment (issaclin32) [reply bucket/revouninstaller.json]: Sorry. Forgot to update that.

**Comment by issaclin32** [comment]
RevoUninstaller shows an error when loading `ctrlbars.dat`, `un_report.dat`, and `winapp_report.dat`.
They are all binary files that cannot be directly interpreted.
Need manual persist.

**Review by rashil2000 — APPROVED** [review]
Tested locally 👍

Just to double-check, the Start Menu shortcut requires UAC permission. Is this expected?

**Comment by issaclin32** [comment]
> Just to double-check, the Start Menu shortcut requires UAC permission. Is this expected?

It is expected. The main program requires admin rights.

**Pull request merged**